pygeoapi/linked_data.py

```diff
@@ -188,16 +188,16 @@ def geojson2jsonld(config, data, dataset, identifier=None, id_field='id'):
         :returns: string of rendered JSON (GeoJSON-LD)
     """
     context = config['resources'][dataset].get('context', []).copy()
+
     defaultVocabulary = {
         'schema': 'https://schema.org/',
-        id_field: '@id',
```

Comment:
Sorry, I've been out of this space for a while, so this may be a dumb comment, but why are we enforcing the use of '@id'? I thought the purpose of this was to allow an existing data source to use whatever property it liked as an effective `@id` without having to literally call it that? Essentially making otherwise naive data somewhat context-aware by laying a minimal LD vocabulary on top without requiring any adjustment to data. Or is this what templating is solving?